src/Types/AccountAddress.cs

```diff
@@ -155,7 +155,16 @@ public AccountAddress GetNthAlias(uint n)
     public override bool Equals(object? obj) =>
         obj is not null && obj is AccountAddress other && this.Equals(other);
 
-    public override int GetHashCode() => this.GetBytes().GetHashCode();
+    public override int GetHashCode()
```

Comment:
The former one is a bug - the hash is taken on the reference pointer of the byte array. Hence the value would be different.
I have already written this in Viktor’s PR and asked him to update these all places where IEquatable interface is overwritten. I needed it here since I used the struct in a HashSet.